Formula/wasmer.rb

```diff
@@ -13,13 +13,12 @@ class Wasmer < Formula
     sha256 cellar: :any_skip_relocation, mojave:        "ca3bb35344bf5e3269e123842f622180939b3b240f0e98f1824ca79305212cae"
   end
 
-  depends_on "cmake" => :build
+  depends_on "llvm" => :build
   depends_on "rust" => :build
-  depends_on "wabt" => :build
 
   def install
     chdir "lib/cli" do
```

Comment:
Try adding
```ruby
ENV.remove "HOMEBREW_LIBRARY_PATHS", Formula["llvm"].opt_lib
```
just before this line to avoid linkage with `llvm`'s `libc++`. Not a guarantee, but it might help.

If you add this change, squash your commits together, please. And use homebrew/core style for the commit message header: we put the name of the formula first in commit-message headings. Refer to the [documentation](https://docs.brew.sh/Formula-Cookbook#commit) for more details.

Thanks!

Reply:
Will try your suggestion and then squash after if everything works! Thanks!